### PR TITLE
Supermarket logos

### LINKED ISSUE #46: Supermarket display
# User Story
As a **user**,
I want to **see supermarket overview**,
so I can **find the cheapest selection**.

# Description

#Material

# Done

- [x] Implement Styling
- [x] Add Icons
- [x] (...)

### PR DESCRIPTION
**Pull request by LebkuchenCJ**
Fix #46 

## Deployment:
Click on SIGN IN -> BERGFEST -> COMPARE  to see results
https://mega-app2020-supermarke-4esq6q.herokuapp.com/

## Comments:
 - I added some logos to my supermarket display option

image

### CODE REVIEW

File: db.json [hunk 1]
```diff
@@ -81,6 +81,7 @@
       "id": 1,
       "name": "Edeka",
       "rating": 4,
+      "logo": "https://i.imgur.com/u2BhkFc.png",
```
Comment (manuelsanchez2) [inline db.json:84]: Cool!

**Review by manuelsanchez2 — APPROVED** [review]
All good! Good job Jonas, I wonder how everything would look if all the logos are horizontal. If you feel like, maybe you can maybe try and see:
[EDEKA](https://www.allibert-outdoor.com/retailer/edeka/edeka-logo/):
[ALDI](https://www.indooroopillyshopping.com.au/media/Indooroopilly/AMP/Store/Aldi-Logo-240-X-200-(3).png?quality=70): 
[LIDL](https://1000logos.net/wp-content/uploads/2020/02/Lidl-logo.png)

Or maybe just change the Rewe one so that it does not differ that much from the others:
[Rewe](https://upload.wikimedia.org/wikipedia/commons/thumb/3/39/Rewe_-_Dein_Markt_Logo.svg/1200px-Rewe_-_Dein_Markt_Logo.svg.png)

File: client/src/components/SupermarketDisplay.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ SupermarketDisplay.propTypes = {
     name: PropTypes.string,
     matchingProductsFound: PropTypes.number,
     listProductCount: PropTypes.number,
-    totalPrice: PropTypes.number,
+    totalPrice: PropTypes.string,
```
Comment (DrBennemin) [inline client/src/components/SupermarketDisplay.js:22]: idk if it is a good practice to express a price in a string. Perhaps you can find a better was to do it in here: [https://frontstuff.io/how-to-handle-monetary-values-in-javascript](https://frontstuff.io/how-to-handle-monetary-values-in-javascript)

**Review by DrBennemin — APPROVED** [review]
your g2g, sir. Just a little thing to think about 😗

File: client/src/components/SupermarketDisplay.js (hunk 2 above)
Comment (LebkuchenCJ) [reply client/src/components/SupermarketDisplay.js:22]: The price is a string as it includes the € symbol as format

**Pull request merged**